enumfields/compat.py

```diff
@@ -65,3 +65,19 @@ def import_string(dotted_path):
         module_path, class_name = dotted_path.rsplit('.', 1)
         module = import_module(module_path)
         return getattr(module, class_name)
+
+
+def import_class(components):
```

Comment:
@akx And also, since we removed the `compat.py` file and I'm trying to rebase this PR to the latest master, I don't know where to put this customized `import_class` function...